templates/geosafe/analysis/dynamic_scripts/create/main.html

```diff
@@ -332,9 +332,6 @@
 
             $.post(analysis_url, post_data, function (data) {
                 if (data) {
-                    area = data['area'] / 1000000;
-                    formatted_area = numeral(area).format('0,0');
```

Comment:
please don't remove these 2 lines. We will figure out how to calculate extent using selected aggregation later.